loads/fetchconf.py

```diff
@@ -1,12 +1,15 @@
 import os
 import json
 import requests
+from outlawg import Outlawg
 
 from loads import (
     URL_GITHUB_RAW,
     DIR_TEMP
 )
 
+Log = Outlawg()
+
 
 # https://raw.githubusercontent.com/rpappalax/dummy-app-01/master/loads.json
 def url_manifest(owner, repo, branch='master'):
```

Comment:
we should eventually allow users to specify a branch too.  something like:
$ loads-cli -l rpappalax/dummy-app-01:dev